pkg/controllers/management/authprovisioningv2/machineconfigs.go

```diff
@@ -0,0 +1,79 @@
+package authprovisioningv2
+
+import (
+	"fmt"
+	"strings"
+
+	v32 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	"github.com/rancher/rancher/pkg/controllers/management/rbac"
+	"github.com/rancher/wrangler/pkg/data"
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+const ownerBindingsAnno = "ownerBindingsCreated"
```

Comment:
nit:
The same constant is already declared at `pkg/controllers/management/nodetemplate/nodetemplate.go`, should we just use that?

Reply:
Yes, we should use that.